main.py

```diff
@@ -13,6 +11,19 @@ async def read_root():
     return {"hello": "world",  "date": now}
 
 
+#code to insert timestamp into bigquery table
+@app.get("/bq_insert_timestamp")
+async def read_root():
+    now = datetime.now()
+    my_dict = {'Timestamp': str(now)}
+    row_to_insert = [my_dict]
+    bq_client = bigquery.Client()
+    table = bq_client.get_table("{}.{}.{}".format("rbfa-workshop-sandboxes", "timestamps_milan", "timestamps"))
```

Comment:
nice to use format, but it has to be a specific case.

In this one the f-string formatting is optimal

Reply:
done